### ISSUE TITLE
Terminal won't open

### ISSUE THREAD

**Issue by Mikestriken** [apommel/vscode-matlab-interactive-terminal#54]
Everytime I select the terminal I get:
'c:\Users\...\.vscode\extensions\apommel.matlab-interactive-terminal-0.4.0\interfaces\unicode\ml_terminal.py' terminated with exit code: 9009.

Python is in my PATH variable:
 → D:\Programs\Miscellaneous\Python\Python310\Scripts\
 → D:\Programs\Miscellaneous\Python\Python310\

What am I doing wrong?

**Comment by Houndoom96** [comment]
It may be because MATLAB Engine API for Python does not support Python 3.10

**Comment by Mikestriken** [comment]
Yes and no; it's mainly because I'm retarded and didn't read the 'requirements' section....

Here are the versions of python that matlab supports for x version of matlab
https://www.mathworks.com/content/dam/mathworks/mathworks-dot-com/support/sysreq/files/python-compatibility.pdf

AND in the requirements section this project currently only supports:
Python x64 2.7, 3.7 or 3.8

So since I'm using the latest version of MatLab I HAVE to use version 3.8 of python.
The latest version of 3.8 with an installer is 3.8.10
soo this version:
https://www.python.org/downloads/release/python-3810/

Then I didn't read the part about installing the MatLab Engine API... well I did I just figured it was something I had to check in the MatLab installer (and not a python file I had to run):
https://www.mathworks.com/help/matlab/matlab_external/install-the-matlab-engine-for-python.html
